markdownlint/TOP009_lessonOverviewItemsSentenceStructure/TOP009_lessonOverviewItemsSentenceStructure.js

```diff
@@ -0,0 +1,82 @@
+// A bullet list is every token between and inclusive bullet_list_open to bullet_list_closed
+function isolateBulletList(tokens) {
+  const bulletList = [];
+  let inBulletList = false;
+  let inLessonOverview = false;
+  tokens.forEach((token, index, tokensArr) => {
+    if (
+      tokensArr[index - 1]?.type === "heading_open" &&
+      /lesson overview/i.test(token.content)
+    ) {
+      inLessonOverview = true;
+    } else if (token.type === "bullet_list_open") {
+      inBulletList = true;
+    } else if (tokensArr[index - 1]?.type === "bullet_list_close") {
+      inLessonOverview = false;
+      inBulletList = false;
+    }
+
+    if (inBulletList && inLessonOverview) bulletList.push(token);
+  });
+  // Only consider tokens that have text in them.
+  return bulletList.filter((token) => token.type === "inline");
+}
+
+// A list item is every token between and inclusive list_item_open to list_item_close
+function getListItemData(bulletPoint) {
+  const lineNumber = bulletPoint.lineNumber;
+  const context = bulletPoint.content.trim();
+  const firstCharacter = context.at(0);
+  const lastCharacter = context.at(-1);
+  return {
+    firstCharacter,
+    lastCharacter,
+    lineNumber,
+    context,
+  };
+}
+
+module.exports = {
+  names: ["TOP009", "lessonOverviewItemsSentenceStructure"],
```

Comment:
```suggestion
  names: ["TOP009", "lesson-overview-items-sentence-structure"],
```
Kebab case for consistency between other rules (the lint report shows the rule name by doing `.names.join("/")`